pmm/v3/pmm3-release-candidate.groovy

```diff
@@ -137,7 +137,7 @@ pipeline {
             name: 'REMOVE_RELEASE_BRANCH'
         )
         string(
-            defaultValue: '#pmm-dev',
+            defaultValue: '#pmm',
```

Comment:
information about RC started should go to pmm-internal instead of pmm

Reply:
Thanks, I missed that one.